scripts/globals/magic.lua

```diff
@@ -32,9 +32,37 @@ xi.magic.singleWeatherWeak     = { xi.weather.RAIN,              xi.weather.HOT_
 xi.magic.doubleWeatherWeak     = { xi.weather.SQUALL,            xi.weather.HEAT_WAVE,        xi.weather.BLIZZARDS,          xi.weather.GALES,              xi.weather.SAND_STORM,             xi.weather.THUNDERSTORMS,       xi.weather.DARKNESS,        xi.weather.STELLAR_GLARE  }
 xi.magic.eemStatus             = { xi.effect.FIRE_EEM_MOD,       xi.effect.ICE_EEM_MOD,       xi.effect.WIND_EEM_MOD,        xi.effect.EARTH_EEM_MOD,       xi.effect.THUNDER_EEM_MOD,         xi.effect.WATER_EEM_MOD,        xi.effect.LIGHT_EEM_MOD,    xi.effect.DARK_EEM_MOD    }
 
-xi.magic.eem                   = { 0.05, 0.10, 0.15, 0.20, 0.25, 0.30, 0.40, 0.50, 0.60, 0.70, 0.85, 1.00, 1.15, 1.30, 1.50 }
-xi.magic.eemTiers              = { { eem = 1.50, tier = -3 }, { eem = 1.30, tier = -2 }, { eem = 1.15, tier = -1 }, { eem = 1.00, tier = 0  }, { eem = 0.85, tier = 1  }, { eem = 0.70, tier = 2  }, { eem = 0.60, tier = 3  }, { eem = 0.50, tier = 4  }, { eem = 0.40, tier = 5  }, { eem = 0.30, tier = 6  }, { eem = 0.25, tier = 7  }, { eem = 0.20, tier = 8  }, { eem = 0.15, tier = 9  }, { eem = 0.10, tier = 10 }, { eem = 0.05, tier = 11 } }
-xi.magic.effectEva             = { [xi.effect.SLEEP_I] = xi.mod.SLEEP_MEVA, [xi.effect.SLEEP_II] = xi.mod.SLEEP_MEVA, [xi.effect.POISON] = xi.mod.POISON_MEVA, [xi.effect.PARALYSIS] = xi.mod.PARALYZE_MEVA, [xi.effect.BLINDNESS] = xi.mod.BLIND_MEVA, [xi.effect.SILENCE] = xi.mod.SILENCE_MEVA, [xi.effect.PLAGUE] = xi.mod.VIRUS_MEVA, [xi.effect.PETRIFICATION] = xi.effect.PETRIFY_MEVA }
+xi.magic.eemTiers =
+{
+    { eem = 1.50, mult = 0.95,    tier = -3 },
+    { eem = 1.30, mult = 0.96019, tier = -2 },
+    { eem = 1.15, mult = 0.98,    tier = -1 },
+    { eem = 1.00, mult = 1,       tier = 0  },
+    { eem = 0.85, mult = 1.023,   tier = 1  },
+    { eem = 0.70, mult = 1.049,   tier = 2  },
+    { eem = 0.60, mult = 1.0905,  tier = 3  },
+    { eem = 0.50, mult = 1.126,   tier = 4  },
+    { eem = 0.40, mult = 1.2075,  tier = 5  },
+    { eem = 0.30, mult = 1.3475,  tier = 6  },
+    { eem = 0.25, mult = 1.70065, tier = 7  },
+    { eem = 0.20, mult = 2.141,   tier = 8  },
+    { eem = 0.15, mult = 2.65,    tier = 9  },
+    { eem = 0.10, mult = 2.96,    tier = 10 },
+    { eem = 0.05, mult = 3.52,    tier = 11 }
+}
+
+xi.magic.effectEva =
+{
+    [xi.effect.SLEEP_I] = xi.mod.SLEEP_MEVA,
```

Comment:
Slow?

Reply:
I believe we are also missing Amnesia, Virus, Gravity, Stun, Sleep (Light), Charm, Bind, Terror,

Reply:
These are only for barspells guys. This covers all of them. :X